Cauli/Configuration/Configuration.swift

```diff
@@ -34,7 +37,13 @@ public struct Configuration {
     public static let standard = Configuration(
         recordSelector: RecordSelector.max(bytesize: 5 * 1024 * 1024),
         enableShakeGesture: true,
-        storageCapacity: .records(50))
+        storageCapacity: .records(50),
+        prePersistHook: { record in
```

Comment:
Kept that in while this PR is in WIP to allow for easier testing. This needs to be removed before the PR is merged.

Reply:
- [ ] Remove this before merging.

Reply:
🙃 you could also try TDD the next time. Then we already have a test case for our test suit.